.circleci/config.yml

```diff
@@ -163,11 +208,18 @@ workflows:
               python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11" ]
           requires:
             - ci-base-<< matrix.python-version >>
+      - integration-test-parallel:
+          matrix:
+            parameters:
+              python-version: [ "3.9" ]
+              pytest-group: [ detection, fr, generic ]
+          requires:
+            - ci-base-<< matrix.python-version >>
       - integration-test:
           matrix:
             parameters:
               python-version: [ "3.9" ]
-              pytest-group: [ detection, classification, fr, generic, misc ]
```

Comment:
Any reason not to parallelize `classification` and `misc` too, removing the need for separate `integration-test` and `integration-test-parallel` job definitions?

Reply:
could revisit and see.